### PR TITLE
build: upgrade to @hapi/hapi

### LINKED ISSUE #1215: support for latest @hapi/hapi v18+
# Feature Request

#### What is the desired behavior?
Support HAPIJS SSR via the newly located `@hapi/hapi` npm package... `hapi` is deprecated
https://github.com/angular/universal/blob/master/modules/hapi-engine/package.json#L15

#### What is the use-case or motivation for the desired behavior?
`hapi` is deprecated

#### Is there anything else we should know?
you can add an alias to easily support the new location

https://twitter.com/hapijs/status/1120237590668386304

### PR DESCRIPTION
**Pull request by CaerusKaru**
Fixes #1215

### CODE REVIEW

**Comment by CaerusKaru** [comment]
Blocked on https://github.com/bazelbuild/rules_nodejs/issues/1033

File: modules/hapi-engine/schematics/install/index.spec.ts [hunk 1]
```diff
@@ -45,13 +45,13 @@ describe('Universal Schematic', () => {
     expect(contents).toMatch(/\"@nguniversal\/hapi-engine\": \"/);
   });
 
-  it('should add dependency: hapi', async () => {
+  it('should add dependency: @hapi/hapi', async () => {
```
Comment (alan-agius4) [inline modules/hapi-engine/schematics/install/index.spec.ts]: Missed a commit to `modules/hapi-engine/schematics/install/index.ts`?
https://github.com/angular/universal/blob/fa6d235a5460dd8ff4d2caa69227eaf26bc3d6ba/modules/hapi-engine/schematics/install/index.ts#L71-L80

Both hapi and inert dependencies need to be changed

**Comment by alan-agius4** [comment]
@CaerusKaru, you should unblocked on this.
See: https://github.com/bazelbuild/rules_nodejs/issues/1033#issuecomment-542820310

File: modules/hapi-engine/schematics/install/index.ts [hunk 2]
```diff
@@ -42,17 +42,22 @@ function addDependencies(options: UniversalOptions): Rule {
...
       type: NodeDependencyType.Default,
-      name: 'inert',
+      name: '@hapi/inert',
       version: '^5.1.0',
     });
     addPackageJsonDependency(host, {
       type: NodeDependencyType.Dev,
-      name: '@types/inert',
+      name: '@types/hapi__inert',
       version: '^5.1.0',
```
Comment (alan-agius4) [inline modules/hapi-engine/schematics/install/index.ts]: This should be `5.2.0`, `5.1.0` is non existing for this package.

Comment (alan-agius4) [reply modules/hapi-engine/schematics/install/index.ts]: Addressed the versions in my fixup commit.

**Comment by CaerusKaru** [comment]
@googlebot I consent.

**Comment by alan-agius4** [comment]
@googlebot I consent

**Pull request merged**